src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -184,221 +191,234 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
   };
 
   return (
-    <DialogBox
```

Comment:
I didn't change anything here just moved all this inside <> </>, so I could add the modal component also

Reply:
Got it, thanks